requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+coverage
+pytest
+pytest-cov
+requests-mock
```

Comment:
leaving these unversioned for now; will add in the version info when the Dockerfile is updated to a newer Python

Reply:
Does pytest-cov bring in its own version of coverage? Or does that need to be a separate requirement? I forget...

Reply:
See line above `pytest`  :)

Reply:
I see it. ;) I was just wondering if that was redundant. Guess not!